packages/astro/src/content/runtime.ts

```diff
@@ -44,15 +44,16 @@ export function createCollectionToGlobResultMap({
 	return collectionToGlobResultMap;
 }
 
-const cacheEntriesByCollection = new Map<string, any[]>();
```

Comment:
This was creating global state making it impossible to run the same fixture test multiple times with different config.